crypto/ledger_test.go

```diff
@@ -1,25 +1,27 @@
 package crypto
 
 import (
+	"fmt"
 	"os"
 	"testing"
 
 	"github.com/stretchr/testify/require"
-
-	tcrypto "github.com/tendermint/tendermint/crypto"
+	tmcrypto "github.com/tendermint/tendermint/crypto"
 )
 
-func TestRealLedgerSecp256k1(t *testing.T) {
+var ledgerEnabledEnv = "LEDGER_ENABLED"
```

Comment:
Hmm, do we want to use the same environment variable here? e.g. we want CI to build with Ledger support, but it can't run the tests that require having a Ledger plugged in